Lib/test/test_pathlib.py

```diff
@@ -21,6 +21,15 @@
 except ImportError:
     grp = pwd = None
 
+try:
+    import pwd
+    all_users = [u.pw_uid for u in pwd.getpwall()]
+except (ImportError, AttributeError):
+    all_users = []
```

Comment:
This looks to be a copy-and-paste from `test_os.py`, correct? If so, can you move this to https://github.com/python/cpython/tree/main/Lib/test/support somewhere?